R/render.R

```diff
@@ -1192,3 +1185,21 @@ resolve_df_print <- function(df_print) {
 #' @keywords NULL
 #' @export
 output_metadata = knitr:::new_defaults()
+
+file_scope_split <- function(input, file_scope_fun) {
+  inputs <- file_scope_fun(input)
```

Comment:
Cosmetic (I feel it is cleaner not to repeat the prefix `file_scope_` in the argument):

```suggestion
file_scope_split <- function(input, fun) {
  inputs <- fun(input)
```

Reply:
I agree. Thanks!